### PR TITLE
Embed Panels to Dashboard

### PR DESCRIPTION
**Pull request by dtuckersuh**
- Tutor Requests left blank as we will work on that component next sprint

- Utilized same theme as rest of application

### CODE REVIEW

File: tutor4points/core/templates/home.html [hunk 1]
```diff
@@ -1,10 +1,72 @@
...
+        <div class="col">
+            <form class="form-inline search-box">
+                <input class="form-control search-field" type="search" placeholder="Search subjects/classes">
+                <button class="btn btn-outline-success search-button" type="submit">Search</button>
+            </form>
+            <div class="dashboard-container">
+                <div class="list-container">
+                    <h2>Suggested Sessions</h2>
+                        <!-- TODO: Refactor repeated code -->
+                        {% for tutor in tutors %}
```
Comment (thivnguyen) [inline tutor4points/core/templates/home.html:36]: The tutors do not appear on the "suggested sessions" page. You might need to pass in the tutors from one of the views or create a view to pass in the data.

**Review by thivnguyen — CHANGES_REQUESTED** [review]
The tutors are not appearing on the home page on my end. It's probably because there are no tutors that are being passed in from the home view. I also suggest maybe commenting out the search bar for now in case we don't have time to implement it. Other than that, the subsections all appear.

File: tutor4points/core/templates/home.html (hunk 1 above)
Comment (dtuckersuh) [reply tutor4points/core/templates/home.html:36]: Oh my changes to the view file didn't appear in the commit. I'll update that and put it in.

File: tutor4points/core/templates/home.html [hunk 2]
```diff
@@ -1,10 +1,72 @@
...
+                <h2>Account Balance: {{ user.total_points }} points</h3>
+            </div>
+            <div class="dashboard-container">
+                <div class="list-container">
+                    <h2>Tutor Requests</h2>
+                </div>
+            </div>
+        </div>
+        <div class="col">
+            <form class="form-inline search-box">
```
[the remark this answers is not in the document]
Comment (dtuckersuh) [reply tutor4points/core/templates/home.html:28]: Sure I can comment that out.

**Review by cathyvu1 — APPROVED** [review]
Everything is there so functionality looks good. I suggest limiting the suggested sessions to 2-3 so it doesn't extend the page too much if there are a lot of tutors and to adjust some of the text sizes for the suggested sessions so that the text isn't squished for each box. Also a minor detail but maybe remove the outside box for the suggested sessions.

File: tutor4points/core/static/css/home.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+.classes-column{
+    font-size:20px;
+}
+
+.times-column{
+    font-size: 18px;
+    width:30%
+}
+
+.request-button{
```
Comment (thivnguyen) [inline tutor4points/core/static/css/home.css:64]: I suggest changing the request button color to purple so it matches the most updated button style of the view tutors page.

**Review by thivnguyen — APPROVED** [review]
Looks good to me. The only suggestion is to change the request button to purple so that it matches the buttons on the most updated styling of the tutor's page.

**Pull request merged**